pkg/hubble/parser/parser.go

```diff
@@ -26,6 +28,8 @@ import (
 	monitorAPI "github.com/cilium/cilium/pkg/monitor/api"
 	"github.com/cilium/cilium/pkg/proxy/accesslog"
 	"github.com/golang/protobuf/ptypes"
+	"google.golang.org/protobuf/types/known/timestamppb"
```

Comment:
nit: Throughout Hubble's codebase, we use `github.com/golang/protobuf/ptypes/timestamp` which is an alias.
It would be nice to keep this consistent.

Reply:
Thanks, will adjust.